content/kubernetes/release-notes/k8s-6-2-10-45-2022-07.md

```diff
@@ -0,0 +1,151 @@
+---
+Title: Redis Enterprise for Kubernetes Release Notes 6.2.10-45 (July 2022)
+linktitle: 6.2.10-45 (July 2022)
+description: Support for REBD specific upgrade policies, memcached type REDBs, and RHEL8 for OpenShift, as well as feature improvements and bug fixes.
+weight: 62
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/release-notes/k8s-6-2-10-45-2022-02.md,
+    /kubernetes/release-notes/k8s-6-2-10-45-2022-02/,   ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.2.10-45 supports the Redis Enterprise Software release 6.2.10 and includes feature improvements and bug fixes.
+
+The key bug fixes and known limitations are described below.
+
+## Images
+
+This release includes the following container images:
+
+* **Redis Enterprise**: `redislabs/redis:6.2.10-129` or  `redislabs/redis:6.2.10-129.rhel8-openshift`
```

Comment:
We publish both `redislabs/redis:6.2.10-129.rhel8-openshift` and `redislabs/redis:6.2.10-129.rhel7-openshift` images for this release (also for Kraken 6.2.10-34 with RS 6.2.10-107).

I believe the instructions to users should be to stick to RHEL7 when upgrading from a RHEL7 cluster, and use RHEL8 for new clusters as well as upgrades from RHEL8. Maybe this can go into the "known limitations" in the Lucy release notes since we do have validation and enforcement that no RHEL7-->RHEL8 upgrade is attempted. But in the Kraken release notes this should remain as a warning since we have no such validation there.

Reply:
@bascar @yuvallevy2 what do you think about that?